src/client/sandbox/shadow-ui.js

```diff
@@ -124,6 +124,32 @@ export default class ShadowUI extends SandboxBase {
         document.getElementsByClassName.toString = () => nativeMethods.getElementsByClassName.toString();
     }
 
+    _getUIStyleSheetsHtml () {
+        var stylesheets = this.nativeMethods.querySelectorAll.call(this.document, 'link.' + SHADOW_UI_CLASS_NAME.uiStylesheet);
+        var result      = '';
+
+        for (var i = 0; i < stylesheets.length; i++)
+            result += stylesheets[i].outerHTML;
+
+        return result;
+    }
+
+    _restoreUIStyleSheets (head, uiStyleSheetsHtml) {
```

Comment:
It's always used with the `this.uiStyleSheetsHtml` as the second parameter. So, do we need this parameter?